packages/api/__tests__/invoke.test.ts

```diff
@@ -0,0 +1,93 @@
+import { Api } from '../src';
+
+const { TEST_WORKSPACE_ID } = process.env;
+
+describe('invoke', () => {
+  let api: Api;
+
+  beforeEach(() => {
+    api = new Api({
+      workspaceId: TEST_WORKSPACE_ID as string,
+    });
+  });
+
+  it('runs specified webhook', async () => {
+    const response = await api.invoke('webhookWithoutParams', {
+      method: 'GET',
+    });
+    const json = await response.json();
+
+    expect(json).toMatchObject({
+      data: null,
+    });
+  });
+
+  it('passes data', async () => {
+    const response = await api.invoke('webhookWithoutParams', {
+      method: 'GET',
+      data: {
+        test: 'some value',
+      },
+    });
+    const json = await response.json();
+
+    expect(json).toMatchObject({
+      data: {
+        test: 'some value',
+      },
+    });
+  });
+
+  it('fetches by path specified', async () => {
+    const response = await api.invoke('webhookWithParams', {
+      method: 'POST',
+      data: {
+        test: 'some value',
+      },
+      path: 'firstParam/path/secondParam/test',
+    });
+    const json = await response.json();
+
+    expect(json).toMatchObject({
+      data: {
+        test: 'some value',
+      },
+      pathParameters: {
+        param1: 'firstParam',
+        param2: 'secondParam',
+      },
+    });
+  });
+
+  it('passes headers', async () => {
+    const response = await api.invoke(
+      'webhookWithParams',
+      {
+        method: 'POST',
+        data: {
+          test: 'some value',
+        },
+        path: 'firstParam/path/secondParam/test',
+      },
+      {
+        headers: {
```

Comment:
Hey @vorobeez is there any reason we need to separate the headers object from the 2nd positional argument?

Reply:
I made it by several reasons.

1. It's like in our `request` method.
2. In the second argument user passes webhook-dependend data. In the third argument user passes fetch-dependent data. He can pass not only headers but all fetch options.